src/server/services/wfs/qgswfs.cpp

```diff
@@ -47,14 +47,16 @@ namespace QgsWfs
 
       /**
        * Constructor for WFS service.
+       * \param version Version of the WFS service.  \since QGIS 3.22.12
```

Comment:
`\since` is strangely positioned no?

Reply:
I do it like @lbartoletti proposed it.

Reply:
@lbartoletti is the `\since` has to be for the method or for the parameter ?

Reply:
For the parameter, but it should be `(since xx)` and not `\since xx`, sorry

Reply:
@lbartoletti @pblottiere changes done. Thanks